src/ResourceManager/Automation/Commands.Automation/Common/IAutomationClient.cs

```diff
@@ -291,8 +291,8 @@ public interface IAutomationClient
         IEnumerable<JobSchedule> ListJobSchedulesByRunbookName(string resourceGroupName, string automationAccountName, string runbookName);
 
         IEnumerable<JobSchedule> ListJobSchedulesByScheduleName(string resourceGroupName, string automationAccountName, string scheduleName);
-
-        JobSchedule RegisterScheduledRunbook(string resourceGroupName, string automationAccountName, string runbookName, string scheduleName, IDictionary parameters);
+                
+        JobSchedule RegisterScheduledRunbook(string resourceGroupName, string automationAccountName, string runbookName, string scheduleName, IDictionary parameters, string RunOn);
```

Comment:
I have reviewed it with Safeer. RunOn parameter is optional and wouldnt impact if not passed, right?